MetaCell/salk-interactive-atlas
# Feature/258: Delete Experiment

Closes #258 

Implemented solution: 
- Created a modal for confirmation when the "delete experiment" is clicked from the experiment card menu. 
- This triggers the Delete API to delete the experiment
- Refresh the experiment thereafter 

How to test this PR: 
- From the Menu, delete the experiment 
![image](https://github.com/MetaCell/salk-interactive-atlas/assets/59233227/f75b33b7-51a6-4484-aa6d-1492f8600392)

- The modal pops up
![image](https://github.com/MetaCell/salk-interactive-atlas/assets/59233227/c7d42638-ff06-4d85-9594-141f4447ac61)

- Press delete and check if the experiment has been deleted.



### Sanity checks:

### Breaking changes (select one):

### Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):

### Nice to have (if relevant):
